test/unit/ngsi-ld/lazyAndCommands/polling-commands-test.js

```diff
@@ -153,22 +149,14 @@ describe('NGSI-LD - Polling commands', function() {
         });
     });
 
-    xdescribe('When a command update arrives to the IoT Agent for a device with polling', function() {
+    describe('When a command update arrives to the IoT Agent for a device with polling', function() {
```

Comment:
All the xdescribe() and xit() has been solved in this PR? Or is there any pending after it?

Reply:
There are following PRs - I haven't complete finished.

Reply:
Note that this is because NGSI-LD is using entity upsert for both create and update so the expectations can't just be duplicated from v2.

Reply:
Thanks for the clarification. NTC.